### DISCUSSION TITLE
mount android resources

### DISCUSSION THREAD

**Discussion by BKnova** [DesktopECHO/Pi-hole-for-Android#104] [Q&A]
If this is a silly question, please forgive me.
Main purpose: I want NextCloudPi to use an external USB hard drive connected to an Android phone. Can Pi Deploy mount an external hard drive? Is this possible?
Device info: Redmi Note 9T; MIUI Global 14.
I tried using the "lsblk" command in the openssh to find the USB path, but the "MOUNTPOINTS" column does not show any information. and I navigated to mnt/media_rw/ ,but can't find anything.
I'm stuck. Could someone with experience please help me?
![image](https://github.com/user-attachments/assets/abeaeaea-948e-4ea2-ac0e-27b20285019b)
![image](https://github.com/user-attachments/assets/449aa240-177e-4f95-8e9b-6e9cb95e32a6)

8791-D3D9 is my target external USB path in android,
and"7C16F5C816F58404" is Pi Deploy default folder
![image](https://github.com/user-attachments/assets/d28eeb4b-042b-4f36-b491-2ef8ddd5f67f)
mount's setting of Pi Deploy
probably,my mount's setting is wronged.
![image](https://github.com/user-attachments/assets/79efb120-7a6d-483d-bac3-64c13833151d)

Lastly, I would like to say a big thank you to the developers who contributed to Pi-hole-for-Android. Even though I'm not familiar with Raspberry Pi and Linux, I can still enjoy the fun and convenience they bring.

**Comment by DesktopECHO** [comment]
Sorry I missed this message - did you eventually get sorted?
